apps/web/app/auth/callback/route.ts

```
@@ -1,42 +1,32 @@
 import type { NextRequest } from 'next/server'
 import { NextResponse } from 'next/server'
 import { createClient } from '~/lib/supabase/server'
+import { isValidRedirectUrl } from '~/lib/config/validate-url'
+import { AuthError } from '@supabase/supabase-js'
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)